app/prometheus/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+/bin/sed -i 's/{GRAFANA_CLOUD_TOKEN}/'$GRAFANA_CLOUD_TOKEN'/' /etc/prometheus/prometheus.yml
+/bin/sed -i 's/{PROMEHTEUS_ENVIRONMENT}/'$PROMEHTEUS_ENVIRONMENT'/' /etc/prometheus/prometheus.yml
+/bin/prometheus --config.file=/etc/prometheus/prometheus.yml \
+             --storage.tsdb.path=/prometheus \
+             --web.console.libraries=/usr/share/prometheus/console_libraries \
+             --web.console.templates=/usr/share/prometheus/consoles
```

Comment:
What are these `web.console` things?

Reply:
Copied from the original entrypoint of the prom/prometheus image. Not sure what they do, but it is what image would otherwise do if we did not have to make our own.